test/unit/IndividualClaims/redeemClaimPayout.js

```diff
@@ -24,7 +24,7 @@ describe('redeemClaimPayout', function () {
       coverOwner.address,
       0, // productId
       ASSET.ETH,
-      [[coverAmount, timestamp + 1, coverPeriod, 0, false, 0]],
+      [[coverAmount, timestamp + 1, coverPeriod, 7, 0, false, 0]],
```

Comment:
same here use a named constant for the value 7 since it's the same in every test case

Reply:
Yea these are a bit tedious in the case that CoverSegment structure changes. I thought about storing this struct as a fixture somewhere and have each tests just modify which field it is adjusting?

Reply:
yeah agree with fixture